### PR TITLE
Add explaination of addon privacy/security risks

### PR DESCRIPTION
**Pull request by RotationMatrix**
Let me know what your thoughts are on the link to the browser fingerprinting article. Seemed like a good idea at first but it might have been better in my head! 😆 

will-fix: https://dip.torproject.org/web/support/issues/6

### CODE REVIEW

File: content/tbb/tbb-14/contents.lr [hunk 1]
```diff
@@ -6,4 +6,17 @@ seo_slug: installing-add-on-extensions-tor-browser
...
 description:
 It's strongly discouraged to install new add-ons in Tor Browser, because they can compromise your privacy and security.
+
+How?
+Basically, each browser's settings and features create what is called a "browser fingerprint".
+Most browsers inadvertently create a unique fingerprint for each user which can be tracked across the internet.
+Tor Browser is specifically engineered to have a nearly identifcal (we're not perfect!) fingerprint across it's users.
+This means each Tor Browser user looks like every other Tor Browser user, making it difficult to track any individual user.
+
+Now, how does this relate to installing new add-ons in Tor Browser?
```
Comment (gusgustavo) [inline content/tbb/tbb-14/contents.lr]: Let's remove the questions "How?" and "Now, how does this relate to installing new add-ons in Tor Browser" to make reading more fluid.

File: content/tbb/tbb-14/contents.lr [hunk 2]
```diff
@@ -6,4 +6,17 @@ seo_slug: installing-add-on-extensions-tor-browser
...
+Installing new add-ons may affect Tor Browser in unforeseen ways and potentially make your Tor Browser fingerprint unique.
```
Comment (gusgustavo) [inline content/tbb/tbb-14/contents.lr]: This paragraph could go before "Basically, ..." paragraph.

File: content/tbb/tbb-14/contents.lr [hunk 3]
```diff
@@ -6,4 +6,17 @@ seo_slug: installing-add-on-extensions-tor-browser
...
+If your copy of Tor Browser has a unique fingerprint, your browsing activities can be deanonymized and tracked even though you are using Tor Browser.
+
 Tor Browser already comes installed with two add-ons — <mark><a href="https://www.eff.org/https-everywhere">HTTPS Everywhere</a></mark> and <mark><a href="https://noscript.net/">NoScript</a></mark> — and adding anything else could deanonymize you.
+
```
Comment (gusgustavo) [inline content/tbb/tbb-14/contents.lr:23]: Fingerprinting is a great point! 

There's also another points that we could incorporate:

Malicious add-ons: a random add-on installed could compromise your anonymity. Or even non-malicious, they could change your Security settings (NoScript), for example.

Attack surface. Installing new add-ons can increase ways to attack your Tor Browser as the new add-on could leak private information about your OS, for example.

**Review by gusgustavo — COMMENTED** [review]
Feel free to incorporate these topics. The explanation is good!

File: content/tbb/tbb-14/contents.lr (hunk 3 above)
Comment (RotationMatrix) [reply content/tbb/tbb-14/contents.lr:23]: Good point! I forgot about those attack scenarios.

**Pull request merged**